src/app/(home)/_components/hero.tsx

```
@@ -7,80 +7,79 @@ import { getLinks } from 'fumadocs-ui/layouts/shared';
 import Image from 'next/image';
```

Biome (1.9.4)

[error] This import is unused.

Unused imports might be the result of an incomplete refactoring.
Safe fix: Remove the unused import.


(lint/correctness/noUnusedImports)